addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -334,7 +334,7 @@
                             <t t-if="dashboard.number_to_check > 0">
                                 <div class="row">
                                     <div class="col-7 text-left">
-                                        <a type="object" name="open_action" context="{'action_name': 'action_move_journal_line', 'search_default_to_check': True}"><t t-esc="dashboard.number_to_check"/> to check</a>
```

Comment:
to whoever review this PR: I don't know why this line is needed

Reply:
To whoever review this PR: This line is needed because we want to show to the user how many invoices/vendor bills he must check, directly in the dashboard view. Like the Misc journal.